packer/jobs/patient_diagnosis_biosource_biomaterial_export.py

```diff
@@ -1,56 +1,63 @@
-import csv
 import logging
-from zipfile import ZipFile
+
 import requests
 from celery.exceptions import Ignore
-from ..table_transformations.patient_diagnosis_biosource_biomaterial import from_obs_json_to_formatted_pdbb_df
+from ..table_transformations.patient_diagnosis_biosource_biomaterial import from_obs_json_to_export_pdbb_df
+from ..table_transformations.utils import filter_rows
 
-from packer.file_handling import FSHandler
 from packer.task_status import Status
 from ..config import transmart_config
 from ..tasks import BaseDataTask, app
+from ..export import save
 
 logger = logging.getLogger(__name__)
-SEP = '\t'
 
 
 @app.task(bind=True, base=BaseDataTask)
-def patient_diagnosis_biosource_biomaterial_export(self: BaseDataTask, constraint, **custom_name):
+def patient_diagnosis_biosource_biomaterial_export(self: BaseDataTask, constraint, **params):
     """
     Reformat hypercube data into a specific export file.
     Export table with the following IDs: Patient, Diagnosis, Biosource, Biomaterial
 
     :param self: Required for bind to BaseDataTask
     :param constraint: should be in job_parameters.
-    :param custom_name: optional in job_parameters.
+    :param params: optional in job_parameters.
```

Comment:
maybe it would be useful to describe what can be added in params fro this job (here or somewhere else). Otherwise you have to analyze the code to know how to use this job.

Reply:
Yes, it'd be useful. I'll work on this.